docs/docs/external/getting-started/interop/data-sharing.md

```diff
@@ -22,9 +22,14 @@ To test the scenario where `Corda_Network` requests the value of the state (key)
 - (_Make sure the following are running_: Corda network, relay, and driver; Fabric `network1`, relay, and driver)
 - Navigate to the `samples/corda/corda-simple-application` folder.
 - Run the following:
-  ```bash
-  ./clients/build/install/clients/bin/clients request-state localhost:9081 localhost:9080/network1/mychannel:simplestate:Read:a
-  ```
+  * Host Deployment:
```

Comment:
I think we are labeling this differently elsewhere. Let's call it "If Network is Deployed using Local Builds".